docs/manifest-releaser.md

```diff
@@ -180,6 +180,11 @@ defaults (those are documented in comments)
   // absence defaults to https://github.com
   "changelog-host": "https://example.com",
 
+  // set custom formatted commit entry
+  // refer to the default conventional-changelog-writer:
+  // https://github.com/conventional-changelog/conventional-changelog/blob/11195f2822b389f6b5dc7216c4cf079e0b2883e7/packages/conventional-changelog-writer/index.js#L40
+  "commit-partial": "see above",
```

Comment:
Not sure if there is any alternative to dumping the contents of `commit.hbs` into this.

Reply:
Oh I just realised I can probably just provide a file path. That is much better.